flink-runtime-web/web-dashboard/src/app/pages/job/job.module.ts

```diff
@@ -53,7 +56,10 @@ import { NzPopconfirmModule } from 'ng-zorro-antd/popconfirm';
     NzDividerModule,
     NzCollapseModule,
     NzEmptyModule,
+    NzIconModule,
+    NzSelectModule,
     NzSkeletonModule,
+    NzToolTipModule,
```

Comment:
Same goes for the tooltip. Both modules, `NzToolTipModule` and `NzIconModule` were added on `master` in [903185d72c](https://github.com/apache/flink/commit/903185d72c#diff-b076916fff7fedff49389f5a54ec236ef413a1a43bc6a7b95406266d9fb12fe7R64-R66)